src/plugins/mer/meroptionswidget.cpp

```diff
@@ -159,6 +165,46 @@ void MerOptionsWidget::store()
                 ok = false;
             }
         }
+        if (m_memorySize.contains(sdk)) {
+            if (m_memorySize[sdk] != sdk->memorySize()) {
```

Comment:
You mean that the settings are stored in xml and don't updated from VirtualBox?

Reply:
Yes, QtC works with cached values. Users are not expected to fiddle with the VM settings using VBox interfaces. Of course users are naughty at times and for that reason all values are updated unconditionally from qtc side to allow restoring consistency somehow conveniently in case of troubles.

Reply:
What do you mean by every time? After sdk upgrade? That should be handled by the installer. This is also something we will have to have a closer look at later, what happens with these values during upgrade. More fun waiting for us yet :)